### ISSUE TITLE
How to use STN when the batch size of train set and test set is different?

### ISSUE THREAD

**Issue by wangyujie413** [kevinzakka/spatial_transformer_network#22]
The tensorflow is build the grapy first, if the batch size of train set and test set is different, how do I set the value of B?
